microprofile.jdt/com.redhat.microprofile.jdt.test/src/main/java/com/redhat/microprofile/jdt/core/BasePropertiesManagerTest.java

```diff
@@ -187,7 +181,7 @@ public void run(IProgressMonitor monitor) throws CoreException {
 		// property:
 		// deployment-artifact=io.quarkus\:quarkus-hibernate-orm-deployment\:0.21.1
 
-		IJavaProject javaProject = JavaModelManager.getJavaModelManager().getJavaModel().getJavaProject(description.getName());
+		IJavaProject javaProject = JavaModelManager.getJavaModelManager().getJavaModel().getJavaProject(projectName);

```

Comment:
After a discussion with @fbricon, we can stick with Maven tests for now. When loading Maven projects (ie, from VS Code), the IResource path is `P/folderName`, therefore we should use `projectName` (which is the folder name) here instead of `description.getName()`.

This essentially reverts a change that https://github.com/redhat-developer/quarkus-ls/pull/330 did.